main.go

```diff
@@ -307,7 +322,7 @@ func handlerWebhook(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if webhook.Attributes.Action == "open" && mr.ForceRemoveSourceBranch != true {
-		defer setRemoveSourceBranchForMR_AndReport(webhook.Attributes.SourceProjectID, webhook.Attributes.IID)
+		defer setRemoveSourceBranchForMR_AndReport(webhook.Attributes.SourceProjectID, webhook.Attributes.II, webhook.Attributes.SourceBranch)
```

Comment:
If I understand well, you wanted to write `webhook.Attributes.IID` instead of `webhook.Attributes.II`